### DISCUSSION TITLE
A32NX YAW DAMPER KNOB

### DISCUSSION THREAD

**Discussion by Mickepilot** [flybywiresim/aircraft#8847] [Help]
When using A32NX with MSFS, the YAW DAMPER KNOB (on the center pedestal) does not move when using Thrustmaster Airbus Quadrant input. However, the digital indication works CORRECT, showing L or R value, depending on Thrustmaster input.

**Comment by alepouna** [comment]
If you can please convert this to an [issue](https://github.com/flybywiresim/aircraft/issues) report (make sure to check for duplicates as well)
